### DISCUSSION TITLE
How to prevent adding note types?

### DISCUSSION THREAD

**Discussion by homocomputeris** [arianneorpilla/jidoujisho#222] [Q&A] [answered]
I do not use built-in note types but Jidoujisho adds them to AnkiDroid. When I delete the note type, Anki forces full sync which is bothersome.

How can I prevent Jidoujisho from adding the built-in note types to AnkiDroid as the default profile cannot be deleted?

**Comment by arianneorpilla** [comment]
Right now this is something I don't individually intend to support unfortunately.

**Comment by arianneorpilla** [comment, marked as answer]
I don't view this is an issue as you can just simply not use it, and the note type being automatically added is intended as a quality of life feature as the app is meant to cater to people who can't tell heaven from earth when it comes to Anki.

Not having a null state for a profile and a card type to export to reduces complexity and potential problems -- I can't enumerate them out of memory but there's a reason why it is currently the way it is.

As I don't see this is as critical and really you can just omit my card type from perception, I won't work on this but anyone interested is free to work on it and I encourage you to make an issue if you desire.

**Reply by homocomputeris** [reply]
Thanks for the reply.
